### DISCUSSION TITLE
Noticed design inconsistency: Ends of black lines

### DISCUSSION THREAD

**Discussion by kaugummiheld** [mirrorkeydev/Appstract#15] [General]
Hey everyone, 

I'm really glad that multiple people contributing to this project now. And I love seeing the designs which the others came up with.

A small thing though, I've noticed something which is not written in the [contribution guide](https://github.com/mirrorkeydev/Appstract/blob/master/CONTRIBUTING.md) up to now. I always use rounded ends when using black lines in my drafts while at least some person are using sharp/perpendicular ends. 
I'm sure in reality that's no big issue due to the scale icons have on mobile devices, but I would like to discuss what the default here should be and whether we want to add something to the guide. 

I'd appreciate it if @mirrorkeydev would give some statement about this. :)

_PS: A little bit OT_
I did see that github now offers a "discussion" function (at least it was advertised in my fork of this repo). I'd rather use this, since the above is not really an issue but ... well, a discussion. :sweat_smile: 
So I'd be for turning this on here.

**Comment by mirrorkeydev** [comment]
Hi @kaugummiheld, great point!

You're right, rounded caps are the de facto norm for Appstract :) If a couple flat caps slipped through, it's on me for not noticing and not putting it in the contribution guide (TODO for me haha)

Also, I enabled discussions! Feel free to put future discussions in there :)

Edit: Aha! I also figured out how to convert your issue to a discussion :)

**Reply by kaugummiheld** [reply]
Okay, I thought so but wasn't sure about it. Thank you for clarifying.

> Edit: Aha! I also figured out how to convert your issue to a discussion :)

Cool! :)

> not putting it in the contribution guide

I think this could be added relative easily to the [examples picture](https://github.com/mirrorkeydev/Appstract/blob/master/examples.png) or in the picture below where you already have dos and don'ts.
